tests/conftest.py

```diff
@@ -18,22 +17,32 @@ def filepath() -> str:
     assert os.path.exists(path)
     yield os.path.abspath(path)
 
+@pytest.fixture(scope='session')
```

Comment:
Does this reliably work? I think you might want to use a "fixture class" with __init__ and __del__ methods that do construction and teardown. I don't know what the pattern should be here.

Reply:
I think the problem is those are remnants of python unittest. The stuff I was reading said not to mix those two. The pytest way is to either do the temp path, or to have a cleanup portion of the fixture after the `yield` statement